includes/class-bsf-custom-fonts-admin.php

```diff
@@ -133,20 +142,54 @@ public function manage_columns( $columns ) {
 			return $columns;
 		}
 
+		/**
+		 * Add new taxonomy data.
+		 */
+		public function add_new_taxonomy_data() {
+			$this->add_new_taxonomy_default_data();
+			?>
+			<input type="hidden" name="repeater-field-count" value="1">
+			<?php
+			echo '
+			<div id="repeater">
```

Comment:
@premanshup - Shall we make use of output buffer here `ob_start()` , `ob_end_clean()`, etc in order to avoid double/single quote mismatch?
Also, HTML in quotes is tough for readability too.
Let me know what you think @Nikschavan

Reply:
@vrundakansara I have removed rendering HTML and added it directly.